### ISSUE TITLE
No possibility to just use an owner password and permission

### ISSUE THREAD

**Issue by pprazzi** [jsreport/jsreport-pdf-password#1]
I want to prevent users modifying my pdf documents, and just set the "allow printing in high res" protection level

From the standard pdf security settings, this would just mean setting an owner password, and selecting the permission "12"

https://www.lifewire.com/what-is-a-pdf-owner-password-2619254

I don't want to password protect the document itself (see "How Do I Stop Someone From Opening a PDF?" in that same document)

This doesn't seem to be possible at all with jsreport-pdf-password, I've already tried from within jsreport-studio and directly by calling the web api. In the first it's not possible to select owner passwords without main passwords, and in the second it just doesn't do anything if I just set ownerPassword and protectionLevel

**Comment by bjrmatos** [comment]
hi! i've published `v1.1.0` where we allow setting owner password and protection level without the main password, however as you say when you specify only the owner password and protection level nothing happens, this is because a limitation on the [library we are using to encrypt the pdf](https://github.com/galkahana/HummusJS), which doesn't encrypt the file without the main password. 

you can help us opening an issue on the [library](https://github.com/galkahana/HummusJS) asking support for this.

**Comment by pprazzi** [comment]
Thanks! Did you create an existing issue for this on HummusJS so I can add my comment(s)/vote to it?

**Comment by bjrmatos** [comment]
@pprazzi no, i didn't have created an issue yet, go ahead and open one and feel free to ping/cc me 😄

**Comment by rvidya** [comment]
Hi @pprazzi and @bjrmatos , Did the discussed issue come to a resolution? I also am trying to edit-proof my js-report. If you can give any pointers, that would be very helpful.

**Comment by ilyavf** [comment]
+1
